source/declarations.tex

```diff
@@ -2401,6 +2409,11 @@
 \grammarterm{type-id}
 in its syntactic context shall be considered a
 \grammarterm{type-id}.
+However, a construct that can syntactically be a \grammarterm{type-id}
+whose outermost \grammarterm{abstract-declarator}
+would match the grammar of an \grammarterm{abstract-declarator}
+with a \grammarterm{trailing-return-type}
+is to be considered a \grammarterm{type-id} only if it starts with \keyword{auto}.
```

Comment:
I'm using <https://isocpp.org/files/papers/P2915R0.pdf> to review.
It actually says
```suggestion
is a \grammarterm{type-id} only if it starts with \keyword{auto}.
```
which is more consistent with the other additions.

Reply:
I disagree - it makes no sense for something to *be* a grammarterm. A grammarterm names a production rule, a purely lexical construct - it *is* its rule. "is to be considered" isn't right either, but at least we're not trying to redefine a rule.

Reply:
Fixed original commit to follow paper (as suggested) and added "to be considered" in a separate commit.